scm/driver/gitlab/content.go

```diff
@@ -19,23 +19,12 @@ type contentService struct {
 }
 
 func (s *contentService) Find(ctx context.Context, repo, path, ref string) (*scm.Content, *scm.Response, error) {
-	path = url.QueryEscape(path)
-	path = strings.Replace(path, ".", "%2E", -1)
```

Comment:
This had been dropped without a replacement? Not sure if `url.QueryEscape` is really doing that?

Reply:
Oh yeah I should've added some more comments. The GitLab documentation says:
```
file_path (required) - Url encoded full path to new file. Ex. lib%2Fclass%2Erb
```
Which is arguably ambiguous about the meaning of "Url encoded," since dots are unreserved characters according to [RFC3986](https://tools.ietf.org/html/rfc3986#section-2.3).
And despite that the example in GitLab's documentation used `%2E`, the API server responded properly even if the dot is not replaced. So this replace is unnecessary.

That said, if you think it is safer to escape the dots, I'm happy to revert this change.

Reply:
I don't know how often this had been changed within the Gitlab API, as we are not only focussing on the hosted version it's possible that older Gitlab versions act differently. In the end it's up to brad if he's fine with dropping this encoding or if he wants to keep it as he will be hit by some possible support burden.

Reply:
I would ask that we revert changes made to this method (both to encoding and changes to the endpoint).  Let's keep this pull request focused on `List` and we can discuss changes to this method in a separate pull request.